src/Transports.AspNetCore/WebSockets/GraphQLWs/MessageType.cs

```diff
@@ -0,0 +1,155 @@
+using GraphQL.Transport;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets.GraphQLWs;
+
+/// <summary>
+/// Protocol message types defined in
+/// <see href="https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md"/>
+/// and
+/// <see href="https://github.com/enisdenjo/graphql-ws/blob/master/src/common.ts"/>
+/// </summary>
+public static class MessageType
+{
+    /// <summary>
+    /// <para>
+    /// Indicates that the client wants to establish a connection within the existing socket.
+    /// This connection is not the actual WebSocket communication channel, but is rather a
+    /// frame within it asking the server to allow future operation requests.
+    /// </para>
+    /// <para>
+    /// The server must receive the connection initialization message within the allowed waiting
+    /// time specified in the <see cref="GraphQLWebSocketOptions.ConnectionInitWaitTimeout"/>
+    /// parameter during the server setup.  If the client does not request a connection within
+    /// the allowed timeout, the server will close the socket with the event:
+    /// <i>4408: Connection initialization timeout</i>.
```

Comment:
initiali**s**ation in the original spec 😕

Reply:
I probably fixed any other obvious language flaws in the text but I think I left most of the text the same.  I can't remember.

Reply:
ok